numba_cuda/numba/cuda/decorators.py

```diff
@@ -81,6 +81,14 @@ def jit(
         msg = _msg_deprecated_signature_arg.format("bind")
         raise DeprecationError(msg)
 
+    if isinstance(inline, bool):
+        DeprecationWarning(
+            "Passing bool to inline argument is deprecated, please refer to "
+            "numba's documentation for valid inputs."
```

Comment:
It takes a bit of time to find the documentation and be sure it's the right place, so I think it might help to give users a URL:

```suggestion
            "Numba's documentation on inlining: "
            "https://numba.readthedocs.io/en/stable/developer/inlining.html"
```